internal/hooks/hooks.go

```
@@ -230,6 +402,29 @@ func (m *Manager) RemoveReviewScope(id string) error {
 }
 
+// WriteWhole leaves the file complete or absent, never half written, so
+// a reader polling for it never picks up a partial line. Each writer
+// stages under a name of its own, so two of them cannot publish each
+// other's content.
+func WriteWhole(path, content string) error {
+	staging, err := os.CreateTemp(filepath.Dir(path), filepath.Base(path)+".*.part")
+	if err != nil {
+		return err
+	}
+	defer os.Remove(staging.Name())
```

golangci-lint (2.13.2)

[error] Error return value of `os.Remove` is not checked

(errcheck)